DemoApp/DemoApp/Constant/Constants.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DemoApp.Constant
+{
+    internal static class Constants
```

Comment:
There is no need to have these as constants. These are used in the UI. In a typical production application, we should have these in the resource file. But since this is a demo app, we don't have to do that. 
But we should definitely not have these as constants.